docs/CONFIG.md

```diff
@@ -5235,8 +5235,8 @@ BumpMin = '5 gwei'
 BumpPercent = 20
 BumpThreshold = 0
 EIP1559DynamicFees = false
-FeeCapDefault = '100 micro'
-TipCapDefault = '1 wei'
+FeeCapDefault = '1 gwei'
```

Comment:
note I got these values from just asking the new sim SuggestGasPrice SuggestGasTip etc.